### PR TITLE
fixing: renamed `Label` to `Attribute` to align with metric specification

### LINKED ISSUE #591: Rename metrics labels to attributes
See the spec change for details https://github.com/open-telemetry/opentelemetry-specification/pull/1775

### PR DESCRIPTION
**Pull request by LanceEa**
Fixes #591 

* Label is still being used when referring to Prometheus labels but when referring to OpenTelemetry metrics the term `attribute` is now used.
* The `opentelemetry-proto` library still has the `labels` field and will be deprecated approximately 3 months from July 1st 2021. At that point we can remove it when collecting metrics ( `labels: vec!()`)

### CODE REVIEW

**Comment by LanceEa** [comment]
@TommyCpp - I only changed the change log for `opentelemetry` crate since that is where `Attribute` is defined. Let me know if you want me to modify the other crates that were affected as well.

Cheers,
Lance

File: opentelemetry/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v0.16.0](https://github.com/open-telemetry/opentelemetry-rust/compare/v0.15.0...v0.16.0)
+
+### Changed
+
+- Renamed `label` to `attribute` to align with otel specification #591
```
Comment (TommyCpp) [inline opentelemetry/CHANGELOG.md:7]: nit: we usually use the PR id here

**Review by TommyCpp — APPROVED** [review]
Awesome! Just a small nit

File: opentelemetry/CHANGELOG.md (hunk 1 above)
Comment (LanceEa) [reply opentelemetry/CHANGELOG.md]: Pushed a fix  with the updated PR # on the changelog.

**Pull request merged**